playbook.yaml

```diff
@@ -0,0 +1,86 @@
+---
+- name: configure remote vm
+  hosts: Remote
+  remote_user: ec2-user
+  environment: 
+    PATH: /home/ec2-user/.poetry/bin:{{ ansible_env.PATH }}
+
+  vars_prompt:
+    - name: app_token
+      prompt: app_token
+      private: no
+  
+    - name: app_key
+      prompt: app_key
+      private: no
+  
+    - name: not_started_list_id
+      prompt: not_started_list_id
+      private: no
+  
+    - name: started_list_id
+      prompt: started_list_id
+      private: no
+  
+    - name: done_list_id
+      prompt: done_list_id
+      private: no
+
+  tasks:
+  - name: install packages
+    yum:
+      name: 
+        - git
+        - python3
+      state: installed
+    become: yes
+
+  - name: install poetry
+    shell: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+    args:
+      creates: ~/.poetry/bin/poetry
+
+  - name: create directory for todo app
+    file:
+      path: /home/ec2-user/ToDoApp
+      state: directory
+      owner: ec2-user
+      mode: '777'
+    become: yes
```

Comment:
We want to avoid "become: yes" when it's not necessary. You don't need root permissions here (to create a folder inside /home/ec2-user.

You also don't need it for the "copy todo app" task, "poetry install", or "create .env file".